docs/orchestration/flow_config/storage.md

```diff
@@ -248,6 +248,31 @@ which means both upload (build) and download (local agent) times need to have
 the proper Google Application Credentials configuration.
 :::
 
+### Git
+[Git Storage](/api/latest/storage.md#git) is a storage option for referencing flows
+stored in a git repository as `.py` files.
```

Comment:
Add a note on how this differs from other git based storages (ie how it basically supersedes them).